src/nft/components/explore/CarouselCard.tsx

```diff
@@ -223,13 +223,13 @@ export const CarouselCard = ({ collection, onClick }: CarouselCardProps) => {
             <HeaderRow>Uniswap</HeaderRow>
             <HeaderRow>{formatWeiToDecimal(collection.floor.toString())} ETH Floor</HeaderRow>
             <HeaderRow>{gqlCollection.marketplaceCount?.reduce((acc, cur) => acc + cur.count, 0)} Listings</HeaderRow>
-            {MARKETS_TO_CHECK.map((market) => {
+            {MARKETS_TO_CHECK.map((market, index) => {
               const marketplace = gqlCollection.marketplaceCount?.find(
                 (marketplace) => marketplace.marketplace === market
               )
               return (
                 <MarketplaceRow
-                  key={'trendingCollection' + collection.address}
+                  key={`CarouselCard-key-${collection.address}-${index}`}
```

Comment:
this should be keyed on market not index? how can there be duplicates

Reply:
done